### PR TITLE
Document how to build and install for a GitHub release

### PR DESCRIPTION
**Pull request by mprahl**
This excludes instructions for Windows since I don't have a Windows machine handy to test it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,21 +13,53 @@ For more about Open Cluster Management and its Policy Framework:
...
+
+        ```bash
+        mv linux-amd64-PolicyGenerator ${HOME}/.config/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator
+        ```
+
+    - MacOS:
+
+        ```bash
+        mv darwin-amd64-PolicyGenerator ${HOME}/.config/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator
+        ```
```
Comment (ChunxiAlexLuo) [inline README.md:42]: Windows: move by the mouse?

Just kiddinng

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Review by ycao56 — APPROVED** [review]

**Pull request merged**